packages/web3-common/src/eth_execution_api.ts

```diff
@@ -0,0 +1,266 @@
+// The types are generated manually by referring to following doc
+// https://github.com/ethereum/execution-apis
+
+type HexString = string;
+
+// Hex encoded 32 bytes
+type HexString32Bytes = HexString;
+// Hex encoded 1 byte
+type HexStringSingleByte = HexString;
+// Hex encoded 1 byte
+type HexStringBytes = HexString;
+// Hex encoded 256 byte
+type HexString256Bytes = HexString;
+// Hex encoded unsigned integer
+type Uint = HexString;
+// Hex encoded unsigned integer 32 bytes
+type Uint256 = HexString;
+// Hex encoded address
+type Address = HexString;
```

Comment:
These should probably be exported so we can use them elsewhere in the code (like in rpc wrappers)

Also we have [repeats](https://github.com/ChainSafe/web3.js/blob/4.x/packages/web3-utils/src/types.ts#L3)